### PR TITLE
Issue #2494 Renamed backtrack variables to lookahead

### PR DESCRIPTION
**Pull request by dhruvviee**
#2494

### CODE REVIEW

File: Sources/SwiftParser/Statements.swift [hunk 1]
```diff
@@ -208,7 +208,7 @@ extension Parser {
       // However, if this is the first clause, and we see "x = y", then this is
       // almost certainly a typo for '==' and definitely not a continuation of
       // another clause, so parse it as an expression.  This also avoids
-      // lookahead + backtracking on simple if conditions that are obviously
+      // lookahead + lookaheading on simple if conditions that are obviously
       // boolean conditions.
```
Comment (ahoppen) [inline Sources/SwiftParser/Statements.swift:212]:
```suggestion
      // lookahead on simple if conditions that are obviously boolean conditions.
```

**Review by ahoppen — COMMENTED** [review]
Looks like one case slipped through. Otherwise LGTM. Thanks

File: Sources/SwiftParser/Statements.swift (hunk 1 above)
Comment (dhruvviee) [reply Sources/SwiftParser/Statements.swift:212]: Thank you for your suggestion regarding the comments on lines 211 to 212 in Statements.swift. I've made the requested modification as per your recommendation.

**Comment by ahoppen** [comment]
Oh, one last thing. Could you squash your commits? https://github.com/apple/swift-syntax/blob/main/CONTRIBUTING.md#authoring-commits

**Comment by dhruvviee** [comment]
Hi, @ahoppen 
I've squashed the commits as requested. Thank you for your guidance!

**Review by ahoppen — APPROVED** [review]

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by dhruvviee** [comment]
@swift-ci  @ahoppen 
Encountered issue with the test case. I've reviewed the code; can someone please re-run the tests and check the results ?

**Comment by ahoppen** [comment]
Test failures are unrelated. Let’s try again

@swift-ci Please test

**Comment by MaxDesiatov** [comment]
@swift-ci test

**Pull request merged**